### DISCUSSION TITLE
504错误

### DISCUSSION THREAD

**Discussion by lyizhuo** [spencerwooo/onedrive-vercel-index#341] [Q&A] [answered]
<img width="959" alt="image" src="https://user-images.githubusercontent.com/64514694/152106375-f2447389-d596-4d4d-b62f-479da061987b.png">
<img width="630" alt="image" src="https://user-images.githubusercontent.com/64514694/152106447-2f65228f-ca09-4a76-8fc6-836b5a961050.png">

**Comment by spencerwooo** [comment, marked as answer]
我觉得你的 REDIS_URL 没啥问题，可能是因为没重新部署导致环境变量没有生效，试试：

- 重新部署你的 Vercel 站点
- 重新把 Upstash integration 引入 Vercel 中：https://docs.upstash.com/redis/howto/vercelintegration

类似问题：https://onedrive-vercel-index.spencerwoo.com/docs/faqs/error-on-deployment#error-504

**Reply by lyizhuo** [reply]
想请教一下如何重新部署Vercel，是删掉重新导入嘛

**Reply by spencerwooo** [reply]
![image](https://user-images.githubusercontent.com/32114380/152127105-8c21b236-42a8-439d-9785-77825f7651ba.png)
